### ISSUE TITLE
How to reproduce the results.

### ISSUE THREAD

**Issue by tangsanli5201** [mostafa-saad/segment-correction-net#2]
Hi,
   Wonderful idea and work!
  I meet some trouble when trying to reproduce the results.
  My ancilliary model is built and trained by following the setting in the paper and achieves mIoU=85.77% in VOC validation set. However, when I use the Linear Correction to train the primary model, the performance only achieves around 77%.
I follow the settings in the paper as follows:
   \alpha descrease from 30.0 to 0.5.
  Learning rate start from 0.007 with poly decay schedule.
  Stop the gradient at the refined soft labels.
  Any advice?
  Thanks~
